### ISSUE TITLE
Patchface/Shagwell Keeping Keywords with No Source

### ISSUE THREAD

**Issue by teamjimby** [throneteki/throneteki#3318]
I played a game where we both had Patchface and Shagwell in play, and there was another character with insight (Maester Pylos with Glass Candle). I played Florian and Jonquil on Pylos, thereby making him a fool and giving him renown (giving those keywords to the Patchfaces and Shagwells with gold). Florian and Jonquil left play, so Pylos was no longer a fool. However, Patchface/Shagwell kept their keywords. In fact, they kept the keywords for the rest of the game, even though no other characters had insight or renown. This seems to disagree with the ruling on Shagwell.

tldr; Patchface and Shagwell are giving each other keywords via circular logic after other fools are no longer in play.

**Comment by Patane97** [comment]
Duplicate of #3299
